app/controllers/admin/accounting/transactions_controller.rb

```diff
@@ -7,7 +7,8 @@ def index
 
     @transactions = Accounting::Transaction.all
   rescue
-    flash.now[:error] = 'Error connecting to quickbooks'
+    flash.now[:error] = "Error connecting to QuickBooks. Visit
+      #{view_context.link_to('Settings', admin_settings_path)}.".html_safe
```

Comment:
One thing: you can avoid use of html_safe, I think, if you do like this:

`"Error connecting to QuickBooks. Visit " << view_context.link_to('Settings', admin_settings_path)`